docs/whats_new.md

```diff
@@ -79,7 +79,7 @@ batch_input: BatchedInput = {
 
 Where `text` is a batch of texts and `list[images]` is a batch for that texts. This e.g. allows markdown documents with multiple figures like so:
 
-> As you see in the following figure [figure 1](image_1) there is a correlation between A and B.
+> As you see in the following figure figure 1 there is a correlation between A and B.
```

Comment:
Note: This is meant to be an example snippet, not an actual reference to an "image_1"